### ISSUE TITLE
Runtime error when editing a group

### ISSUE THREAD

**Issue by datadavev** [CDLUC3/ezid#180]
```
RuntimeError at /admin/ezidapp/group/1/change/
No active exception to reraise
Request Method:	POST
Request URL:	http://localhost:8000/admin/ezidapp/group/1/change/
Django Version:	3.2.10
Exception Type:	RuntimeError
Exception Value:	
No active exception to reraise
Exception Location:	/Users/vieglais/Documents/Projects/ezid/ezid_py3/impl/log.py, line 328, in otherError
```
The culprit appears to be the `onCommitWithSqliteHack` hack in admin.py: 
```
/Users/vieglais/Documents/Projects/ezid/ezid_py3/ezidapp/admin.py, line 763, in <lambda>
            obj.save()
            sg.save()
        # Our actions won't take effect until the Django admin's
        # transaction commits sometime in the future, so we defer clearing
        # the relevant caches.  While not obvious, the following calls
        # rely on the django-transaction-hooks 3rd party package.  (Django
        # 1.9 incorporates this functionality directly.)
        onCommitWithSqliteHack(lambda: createOrUpdateGroupPid(request, obj, change))
        # Changes to shoulders and Crossref enablement may trigger
        # adjustments to users in the group.
        if change:
            doUpdateUserPids = False
            # Though the group object has been saved above, its shoulder set

```

**Comment by datadavev** [comment]
Seems to be resolved with #175

**Comment by datadavev** [comment]
Appears to be resolved by #175 and bootstrapping operation to prepare PIDs for users and groups
